src/Distributions.cpp

```diff
@@ -4,6 +4,8 @@
 
 #include "Algorithms.cpp"
 
+#define INF std::numeric_limits<double>::max()
```

Comment:
@diego-urgell instead of defining your own INF, you could use INFINITY which is defined in math.h. that would probably be easier to understand for most C programmers who are familiar with INFINITY.